stdlib/theories/Compat/README.md

```diff
@@ -1,9 +1,9 @@
-Files in this directory are intended to be loaded with the `-compat`
-command line option, in order to provide compatibility features to
-mimick some behaviors of older versions. For instance, this can
-disable warnings introduced in later versions.
+Files in this directory are intended to be loaded with the
+`-compat-from Stdlib` command line option, in order to provide
```

Comment:
Which commit is the `-compat-from` flag implemented in?  (It's not the "[stdlib] Adapt Compat" commit)

Reply:
It was part of https://github.com/coq/coq/pull/19370

Reply:
What this PR does is to duplicate the compat infrastructure, by relying on the new `-compat-from` flag that was introduced in #19370 and can be used by any library. Note that `-compat-from` takes two arguments, so it will be `-compat-from Stdlib Stdlib820`. Compatibility for Coq core will still use the `Stdlib` prefix (because `rocq-init` does) but it will be `-compat-from Stdlib Coq820`, or as a shortcut `-compat 8.20` as before.